cmd/parca-agent/main.go

```diff
@@ -276,13 +266,13 @@ func main() {
 
 			w.Header().Set("Content-Type", "application/vnd.google.protobuf+gzip")
 			w.Header().Set("Content-Disposition", "attachment;filename=profile.pb.gz")
-			err = profile.Write(w)
```

Comment:
I think this was important for the download feature on the `Show Profile` page.

Reply:
oops, will add this back.